spec/feature/remember_me_spec.rb

```diff
@@ -98,6 +103,15 @@ def visit_security_page
     click_on I18n.t("navigation.menu_bar.security.link_text")
   end
 
+  def visit_change_email_page
+    within ".accounts-menu" do
+      click_on "Manage your account"
+    end
+    within "#main-content" do
```

Comment:
nice use of within, Curious we're using an ID selector for main... I wonder why, I thought we preferred classes unless IDs were strictly neccessary. Might ask @danacotoran about that one.

Reply:
🤷‍♂️  It was like that in the initial tests too